yayson/yayson.d.ts

```diff
@@ -0,0 +1,82 @@
+// Type definitions for yayson v2.0.1
+// Project: https://github.com/confetti/yayson
+// Definitions by: David Wood <https://github.com/Codesleuth>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'yayson' {
+  class Store {
+    sync(obj: Object): void;
```

Comment:
Yep - mistake. It should be returning the thing that was sync'd.
Why would you use `{}` in the parameter list and not `any`?